teacher.rb

```diff
@@ -0,0 +1,14 @@
+require_relative 'person'
+
+class Teacher < Person
+  attr_reader :specialization
+
+  def initialize(age, name = 'Unknown', specialization: '')
+    super(name, age)
+    @specialization = specialization
```

Comment:
- [ ] Please pass the `parent_permission` parameter to the `initialize` method which is then passed to the `super` method.